package.json

```diff
@@ -16,10 +16,11 @@
   },
   "scripts": {
     "prepublish": "npm run build",
-    "test": "jest",
-    "lint": "tslint 'src/**/*.{ts,tsx}' --project ./",
+    "test": "node node_modules/jest/bin/jest",
```

Comment:
what's the reason behind these changes?

Reply:
and the other script changes etc.

Reply:
I did not realize that npm can figure out local scripts, so actually, this commit isn't needed.